packages/blob-reader/helper.js

```diff
@@ -63,40 +61,47 @@ function getPath(el) {
     );
   }
 
-  let ret = $(rootSelector)
-    .filter(function () {
-      return $(this).text().trim() === 'View file';
-    })
-    .attr('href');
+  const [link] = Array.from(rootSelector).filter((value) => {
+    return value.textContent.trim() === 'View file';
+  });
+
+  let ret = link && link.getAttribute('href');
```

Comment:
There is nothing wrong with this line, but optional chaining might be a fit here 

```suggestion
  let ret = link?.getAttribute('href');
```

Reply:
I keep forgetting this is supported in JS now.

Reply:
@stefanbuck I tried using optional chaining in another PR and Webpack got angry and gave a bunch of errors. Turns out support for them wasn't added until [Acorn 7.3.0](https://github.com/acornjs/acorn/releases/tag/7.3.0) but [Webpack v4 uses Acorn v6.x](https://github.com/webpack/webpack/blob/2efeb4b5787e868e69233aa9dec469c200fa3b2c/package.json#L12) 😞 

According to https://github.com/webpack/webpack/issues/10227#issuecomment-653408484 there's a way to get it to work in Webpack v4 when using Yarn but I haven't tried setting it up yet.